Blockzilla/OverlayView.swift

```diff
@@ -9,24 +10,30 @@ protocol OverlayViewDelegate: class {
     func overlayViewDidTouchEmptyArea(_ overlayView: OverlayView)
     func overlayViewDidPressSettings(_ overlayView: OverlayView)
     func overlayView(_ overlayView: OverlayView, didSearchForQuery query: String)
+    func overlayView(_ overlayView: OverlayView, didSubmitText text: String)
 }
 
 class OverlayView: UIView {
     weak var delegate: OverlayViewDelegate?
-
+    //Settings Button
     fileprivate let settingsButton = UIButton()
-
+    //Search Button
     private let searchButton = InsetButton()
     private let searchBorder = UIView()
     private var bottomConstraint: Constraint!
     private var presented = false
     private var searchQuery = ""
+    
+    //Copy Button
+    private let copyButton = InsetButton()
+    private let copyBorder = UIView()
+
 
     init() {
         super.init(frame: CGRect.zero)
-
```

Comment:
Generally, we should keep PRs as minimal as possible - so removing/adding whitespace should be avoided unless it improves readability. I don't think this changes readability much.